src/libponyrt/platform/ponyassert.c

```diff
@@ -55,7 +58,9 @@ void ponyint_assert_fail(const char* expr, const char* file, size_t line,
       "imprecise or incorrect.\nUse a debug version to get more meaningful "
       "information.\n", stderr);
   }
-
+#else
+  fputs("Backtrace functionality not available.\n", stderr);
```

Comment:
This is the only bit I'm not sure about. Do we want this message?

Marking as "Needs discussion during sync" to get others input. Otherwise this looks good to me.